docs/conf.py

```diff
@@ -48,6 +48,7 @@
   'sphinxcontrib.openapi',
   'sphinxcontrib.httpdomain',
   'sphinx_tabs.tabs',
+  'sphinx_copybutton',
```

Comment:
This module is missing in the requirements. The docs build fails because of that.